spanner/transaction.go

```diff
@@ -151,6 +160,10 @@ func (t *txReadOnly) ReadWithOptions(ctx context.Context, table string, keys Key
 					KeySet:      kset,
 					ResumeToken: resumeToken,
 					Limit:       int64(limit),
+					RequestOptions: &sppb.RequestOptions{
+						RequestTag:     requestTag,
+						TransactionTag: t.qo.transactionTag,
```

Comment:
We don't need transactionTag for readonly transactions.

Reply:
That is true, but this method is used for both read-only and read/write transactions. The `transactionTag` field can however only be set for read/write transactions, so it will always be empty when this is used in combination with a read-only transaction.